pkg/api/v1/conversion.go

```diff
@@ -740,6 +734,7 @@ func Convert_v1_PodSecurityContext_To_api_PodSecurityContext(in *PodSecurityCont
 	return nil
 }
 
+// +k8s:conversion-fn=copy-only
```

Comment:
Correct.  Because defaulting *must* happen, and this is in the gray area between defaulting (a versioned client wants this as well) I moved it out.